internal/iamrolesanywhere/aws_config.tpl

```diff
@@ -1,3 +1,7 @@
 [profile %v]
 region = {{ .Region }}
 credential_process = {{ .SigningHelperBinPath }} credential-process --certificate {{ .CertificatePath }} --private-key {{ .PrivateKeyPath }} --trust-anchor-arn {{ .TrustAnchorARN }} --profile-arn {{ .ProfileARN }} --role-arn {{ .RoleARN }} --role-session-name {{ .NodeName }}{{ if .ProxyEnabled }} --with-proxy{{end}}
+
+[profile hybrid]
```

Comment:
nit: if does admit a comment, can we add one here saying this is for backwards compat and nodeadm doesn't use it?